File: Servers/JS/Fastify/routers/dbSleep.js
const {
  QUERY_SLEEP_0_25,
  QUERY_SLEEP_0_5,
  QUERY_SLEEP_0_75,
  QUERY_SLEEP_1,
  QUERY_SLEEP_2,
} = require("../../db/query");
const { dbSleep } = require("../../utils/paths");
const sleepRouter = async (fastify) => {
  fastify.get(dbSleep.zero25SecondSleep, async () => {
    const client = await fastify.pg.connect();
    try {
      await client.query(QUERY_SLEEP_0_25);
      return { Hello: "World" };
    } finally {
      client.release();
    }
  });
  fastify.get(dbSleep.zero5SecondSleep, async () => {
    const client = await fastify.pg.connect();
    try {
      await client.query(QUERY_SLEEP_0_5);
      return { Hello: "World" };
    } finally {
      client.release();
    }
  });
  fastify.get(dbSleep.zero75SecondSleep, async () => {
    const client = await fastify.pg.connect();
    try {
      await client.query(QUERY_SLEEP_0_75);
      return { Hello: "World" };
    } finally {
      client.release();
    }
  });
  fastify.get(dbSleep.oneSecondSleep, async () => {
    const client = await fastify.pg.connect();
    try {
      await client.query(QUERY_SLEEP_1);
      return { Hello: "World" };
    } finally {
      client.release();
    }
  });
  fastify.get(dbSleep.twoSecondSleep, async () => {
    const client = await fastify.pg.connect();
    try {
      await client.query(QUERY_SLEEP_2);
      return { Hello: "World" };
    } finally {
      client.release();
    }
  });
};
module.exports = sleepRouter;

File: Servers/Python/FastAPI/routers/db_sleep.py
from fastapi import APIRouter, Request

from Servers.Python.utils.db.sql_query import *
from Servers.Python.utils.path import DbSleep

router = APIRouter()


@router.get(DbSleep.zero_25_second_sleep)
async def zero_25_second_sleep(request: Request) -> dict:
    async with request.app.state.pgpool.acquire() as con:
        await con.fetch(
            QUERY_SLEEP_0_25
        )
        return {'hello': 'world'}


@router.get(DbSleep.zero_5_second_sleep)
async def zero_5_second_sleep(request: Request) -> dict:
    async with request.app.state.pgpool.acquire() as con:
        await con.fetch(
            QUERY_SLEEP_0_5
        )
        return {'hello': 'world'}


@router.get(DbSleep.zero_75_second_sleep)
async def zero_75_second_sleep(request: Request) -> dict:
    async with request.app.state.pgpool.acquire() as con:
        await con.fetch(
            QUERY_SLEEP_0_75
        )
        return {'hello': 'world'}


@router.get(DbSleep.one_second_sleep)
async def one_second_sleep(request: Request) -> dict:
    async with request.app.state.pgpool.acquire() as con:
        await con.fetch(
            QUERY_SLEEP_1
        )
        return {'hello': 'world'}


@router.get(DbSleep.two_second_sleep)
async def two_second_sleep(request: Request) -> dict:
    async with request.app.state.pgpool.acquire() as con:
        await con.fetch(
            QUERY_SLEEP_2
        )
        return {'hello': 'world'}
